### ISSUE TITLE
Warn users about about depreciation of python 2.7

### ISSUE THREAD

**Issue by glestel** [nabla-c0d3/sslyze#305]
Hi,

I'm successfully integrating SSLyze with an old tool stuck to python 2.7 and I'm having trouble updating to the last `sslyze 1.4.1` which should should fit the last `1.4.x` main version supported in Python 2.7
This seems to be because of `nassl 1.1.0` that doesn't have a python 2.7 compatible version on linux (if I've well understood how PyPi works). In fact I have the same issue when trying to install `sslyze 1.4.0` on a fresh python 2.7 virtualenv
Following error with SSLyze 1.4.1 :
```
  Downloading SSLyze-1.4.1.tar.gz (1.0MB)
    100% |################################| 1.0MB 956kB/s 
Collecting nassl<1.2.0,>=1.1.0 (from sslyze)
  Could not find a version that satisfies the requirement nassl<1.2.0,>=1.1.0 (from sslyze) (from versions: 0.13.4.win32, 0.13.1, 0.13.2, 0.13.4, 0.13.5, 0.13.6, 0.13.7, 0.14.0, 0.14.1, 0.14.2, 0.15.0, 0.15.1, 0.16.0, 0.16.1, 0.16.2, 0.16.3, 0.17.0, 1.0.1, 1.0.2, 1.0.3)
No matching distribution found for nassl<1.2.0,>=1.1.0 (from sslyze)

```
It runs fine in a python 3.4 virtualenv

**Comment by nabla-c0d3** [comment]
Something is wrong with the nassl Linux wheel for 2.7. All I did was use manylinux to build it (like for the other versions of python) and I don’t think I’ll have the time to debug it.

**Comment by nabla-c0d3** [comment]
I fixed the Linux wheel for Python 2.7 so it works now (https://travis-ci.org/nabla-c0d3/sslyze/jobs/363564491). I will still drop support for Python 2 on the next major release.
